app/backend/src/clj/teet/main.clj

```diff
@@ -40,14 +45,24 @@
               (login-fake-routes/fake-login-routes)))
           (db-api-dev/db-api-routes)
           (routes/teet-routes config))
+         wrap-nil
```

Comment:
why is this needed? isn't 404 automatic if there is no response by any handler

Reply:
Jetty gives you a NPE in this case.